cllc-public-app/ClientApp/src/app/components/security-screening-requirements/security-screening-requirements.component.ts

```diff
@@ -95,7 +95,7 @@ export class SecurityScreeningRequirementsComponent implements OnInit {
   isValid(): boolean {
     this.errorMessages = [];
     let valid = true;
-    if (this.showLiquorContent() && this.applicationId && this.data.liquor.outstandingItems.length > 0) {
+    if (this.showLiquorContent() && this.applicationId && this.data.liquor?.outstandingItems?.length && this.data.liquor?.outstandingItems?.length > 0) {
```

Comment:
The null coalescing operator is not supported in typescript (atleast the version we are using). We can only use the syntax in templates.

Instead of a?.b?.c use a && a.b && a.b.c